docs/documentation/platform/pam/getting-started/resources.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "PAM Resource"
+sidebarTitle: "Resources"
+description: "Learn how to add and configure resources like databases and servers, and set up automated credential rotation."
+---
+
+A resource represents a target system, such as a database, server, or application, that you want to manage access to. Some examples of resources are:
+- PostgreSQL Database
+- MCP Server
+- Linux Server
+- Web Application
+
+## Prerequisites
+
+Before you can create a resource, you must have an **Infisical Gateway** deployed and running on the same network as the target resource.
```

Comment:
It doesn't necessarily have to be in the same network... the resource just has to be reachable by the Gateway